fe/fe-core/src/main/java/org/apache/doris/catalog/ListPartitionInfo.java

```diff
@@ -83,6 +83,12 @@ public PartitionItem createAndCheckPartitionItem(SinglePartitionDesc desc, boole
 
         // we might receive one whole empty values list, we should add default partition value for
         // such occasion
+
+        if (partitionKeyDesc.getInValues() == null) {
+            throw new DdlException(
+                    "Partitions types is invalid, expected In in list partitions");
```

Comment:
Would you please describe the error message more clearly? I don't understand what the root cause of the error is if I only see this error message.

By the way, perhaps checking in the analysis phase would be more suitable.